spec/helpers/link_to_helper_spec.rb

```diff
@@ -146,4 +146,32 @@
 
   end
 
+  describe "when constructing a google analytics event string" do
```

Comment:
Much easier to find the spec you're after when the description is the method name:

```diff
- describe "when constructing a google analytics event string" do
+ describe '#analytics_event_code' do
```